compose/compose-file/compose-versioning.md

```diff
@@ -0,0 +1,301 @@
+---
+description: Compose file reference
+keywords: fig, composition, compose, versions, upgrading, docker
+title: Compose file versions and upgrading
+---
+
+The Compose file is a [YAML](http://yaml.org/) file defining services,
+networks, and volumes for a Docker application.
+
+There are several versions of the Compose file reference available.
+
+- [Version 3](index.md) (most current, and recommended)
+
+- [Version 2 and Version 2.1](compose-file-v2.md)
+
+- [Version 1](compose-file-v1.md)
+
+The topics below explain the differences among the versions, Docker Engine compatibility, and how to upgrade.
+
+## Compatibility matrix
+
+{% include content/compose-matrix.md %}
+
+## Versioning
+
+There are currently four versions of the Compose file format:
```

Comment:
Can we say 3 here? Technically with minor versions there will be 5, but I think we can probably just refer to major versions.